types/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@joystream/types",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

I have already published 2.2.0 so need to bump this one more time.
```suggestion
  "version": "2.3.0",
```